v21.2/create-changefeed.md

```diff
@@ -102,20 +123,72 @@ Option | Value | Description
+`avro_schema_prefix` | Schema prefix name               | Use fully-qualified schema name for a table instead of the default table name. This allows multiple databases or clusters to share the same schema registry when the same table name is present in multiple databases.<br><br>Example: `CREATE CHANGEFEED FOR foo WITH format=experimental_avro, confluent_schema_registry='registry_url', avro_schema_prefix='super'` will register subjects as `superfoo-key` and `superfoo-value` with the namespace `super`.
+`webhook_client_timeout` | [INTERVAL](interval.html)          | <span class="version-tag">New in v21.2:</span> If a response is not recorded from the sink within this timeframe, it will error and retry to connect. Note this must be a positive value. <br><br>**Default:** `"3s"`
+`webhook_auth_header`    | [`STRING`](string.html)            | <span class="version-tag">New in v21.2:</span> Pass a value (password, token etc.) to the HTTP [Authorization header](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Authorization) with a webhook request for a "Basic" HTTP authentication scheme. <br><br> Example: With a username of "user" and password of "pwd", add a colon between "user:pwd" and then base64 encode, which results in "dXNlcjpwd2Q=". `WITH webhook_auth_header='Basic dXNlcjpwd2Q='`.
+`topic_in_value`         | [`BOOL`](bool.html)              | <span class="version-tag">New in v21.2:</span> Set to include the topic in each emitted row update. Note this is automatically set for [webhook sinks](#webhook-sink).
+`webhook_sink_config`    | [`STRING`](string.html)          | <span class="version-tag">New in v21.2:</span> Set fields to configure sink batching and retries. The schema is as follows:<br><br> `{ "Flush": { "Messages": ..., "Bytes": ..., "Frequency": ..., }, "Retry": {"Max": ..., "Backoff": ..., } }`. <br><br>**Note** that if either `Messages` or `Bytes` are nonzero, then a non-zero value for `Frequency` must be provided. <br><br>See the [Webhook sink configuration section](#webhook-sink-configuration) for more details on using this option.
 
 {{site.data.alerts.callout_info}}
  Using the `format=experimental_avro`, `envelope=key_only`, and `updated` options together is rejected. `envelope=key_only` prevents any rows with updated fields from being emitted, which makes the `updated` option meaningless.
 {{site.data.alerts.end}}
 
+#### Webhook sink configuration
+
+<span class="version-tag">New in v21.2:</span> The `webhook_sink_config` option allows the changefeed flushing and retry behavior to your webhook sink to be configured.
+
+The following details the configurable fields:
+
+* `Flush.Messages`: When the batch reaches this configured size, it should be flushed (batch sent).
```

Comment:
This section feels like it might be easier to read as a table, since the list of things about each field appears to be the same